### ISSUE TITLE
IDE1006 false positive for `global` prefix

### ISSUE THREAD

**Issue by huoyaoyuan** [dotnet/roslyn#49535]
**Version Used**: 
VS 16.8.1

**Steps to Reproduce**:

Context: 
https://github.com/microsoft/CsWinRT/blob/cca8f405d87f29d0e2b76f298daa8f8a0640047b/WinRT.Runtime/Projections/Bindable.net5.cs#L1386
```csharp
bool global::System.Collections.IList.IsFixedSize
```
Explicit interface implementation with `global` prefix required.

**Expected Behavior**:
No naming style should be triggered.

**Actual Behavior**:
IDE1006: suggests change to `Global::System.Collections.IList.IsFixedSize`.
